bps/bps_ApTemplate.yaml

```diff
@@ -0,0 +1,47 @@
+# UPDATE THIS to the path to the pipeline to run
+pipelineYaml: '${AP_PIPE_DIR}/pipelines/ApTemplate.yaml'
+
+# Format for job names and job output filenames
+templateDataId: '{tract}_{patch}_{band}_{visit}_{exposure}_{detector}'
+
+# Just names, but UPDATE THIS AND KEEP THEM SHORT
+project: ApTemplate
+campaign: DM-xxxxx
+
+submitPath: ${PWD}/bps/{outCollection}
+computeSite: ncsapool
+# Memory allocated for each quantum, in MBs
+requestMemory: 2048
+# CPUs to use per quantum
+requestCpus: 1
+
+# Example arguments, like the ones you would send to pipetask run from the command line
+payload:
+  runInit: true
+  # UPDATE THIS to also set the output collection name
+  payloadName: DM-xxxxx
+  # UPDATE THIS to point to the correct repository
+  butlerConfig: /repo/dc2/butler.yaml
+  # UPDATE THIS and be sure it includes collections with raws, calibs, refcats, and skymaps
+  inCollection: 2.2i/defaults/test-med-1
+  # Note: keep timestamp in outCollection so you don't get a zillion sub-runs
+  output : 'u/${USER}/{payloadName}'
+  outCollection: '{output}/{timestamp}'
+  # UPDATE THIS to specify what data to process
```

Comment:
Perhaps expand this comment to include the additional keys beyond "band" and "instrument" that people might want to specify.

Reply:
This is a good suggestion, but I have empirically found that doing DECam runs with complex data queries can lead to an extremely lengthy (many days) query time. The example is intentionally minimal, but may be able to be a bit more instructive in the future when this is resolved.